src/utils_format_json.c

```diff
@@ -484,4 +495,215 @@ int format_json_value_list (char *buffer, /* {{{ */
         store_rates, (*ret_buffer_free) - 2));
 } /* }}} int format_json_value_list */
 
+#if HAVE_LIBYAJL
+static int json_add_string (yajl_gen g, char const *str) /* {{{ */
```

Comment:
Sorry, I should have been more specific, I was referring to the difference between `const char *` and `char const *`

I agree that `char const * const *ptr` is indeed much easier to read :)